omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPatientResourceProvider.java

```diff
@@ -182,7 +182,7 @@ public MethodOutcome createFHIRPatient(@ResourceParam Patient patient) {
 		MethodOutcome retVal = new MethodOutcome();
 		retVal.setId(new IdDt(FHIRConstants.PATIENT, patient.getId().getIdPart()));
 		OperationOutcome outcome = new OperationOutcome();
-		outcome.addIssue().setDetails("Patient is successfully created");
+		outcome.addIssue().setDetails("Patient is successfully created with " + patient.getId().getIdPart());
 		retVal.setOperationOutcome(outcome);
 		return retVal;
```

Comment:
Shouldn't you return error responses for patient creation, just like you added for patient updating tasks?